crypto-ffi/src/wasm.rs

```diff
@@ -749,14 +768,28 @@ impl ConversationConfiguration {
         external_senders: Option<Vec<Uint8Array>>,
         key_rotation_span: Option<u32>,
         wire_policy: Option<WirePolicy>,
-    ) -> Self {
+        per_domain_trust_anchors: js_sys::Array,
+    ) -> WasmCryptoResult<ConversationConfiguration> {
         let external_senders = external_senders
             .map(|exs| exs.iter().cloned().map(|jsv| jsv.to_vec()).collect())
             .unwrap_or_default();
-        Self {
+        Ok(Self {
             ciphersuite,
             external_senders,
             custom: CustomConfiguration::new(key_rotation_span, wire_policy),
+            per_domain_trust_anchors: per_domain_trust_anchors
+                .into_iter()
+                .map(|js_anchor| Ok(serde_wasm_bindgen::from_value(js_anchor)?))
+                .collect::<WasmCryptoResult<Vec<_>>>()?,
```

Comment:
As mentioned in the previous comment (on the question you asked) you need to handle the JS `null` here as the value is optional on the WASM side. Even if the wrapper makes sure the value is provided, we'd be safer if it was handled in any case (and as this is a cold path we can pay the price for safety).

```suggestion
            per_domain_trust_anchors: if per_domain_trust_anchors.is_null() {
                vec![]
            else {
                per_domain_trust_anchors
                    .into_iter()
                    .map(|js_anchor| Ok(serde_wasm_bindgen::from_value(js_anchor)?))
                    .collect::<WasmCryptoResult<Vec<_>>>()?
            },
```